CHANGELOG.md

```diff
@@ -5,22 +5,41 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 since 0.4.1, and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [0.7.0] -  2024-06-20
```

Comment:
I understand why now is not the right time to put the date. I would advocate for having it put in this file at some point before we tag, because it's a very useful information for people wanting to upgrade.

I've updated the link with the tag comparison. Good call.